### ISSUE TITLE
BP: Situation Questions

### ISSUE THREAD

**Issue by scottpeterson** [gryphonshafer/Quizzing-Rule-Book#31]
The rule that a maximum # of Situation questions that can be asked per question is 3, is a good rule. 

BUT, there should be an accompanying best practice, that if 3 potential "good" ones exist, you shouldn't write one with just 1 or just 2. 

Now, if there are > 3 "good" situation questions, I'm not sure what the best practice is. Probably want to write 2 situation questions, each with 3 questions on the quotation--but which 3 to include on each? I don't know what's best.

Additionally, when there is a quotation that goes for multiple sentences, you shouldn't write a separate SIT question for each sentence break, unless the division makes sense, flow-wise, and meaning-wise, of the underlying quotations. If them make most sense all together, write a single question. Don't break them up unless it makes sense to do so.

**Comment by jswingle** [comment]
I agree with all the above, but several of the points you made are things that are more about the generation of an overall question _set_ instead of about any particular question. I don't think the best practices document should be giving specific recommendations about question sets - that would be better presented in the context of question writer training instead of any of the documents in this repository. 

I do like the idea of specifying "you should use as many elements of the situation as you think are good to ask, up to three" in the best practices, though.

**Comment by jswingle** [comment]
@scottpeterson I still like this idea, and I think we should write all of the "best practices" concerning SITs at one time so as to be thorough. For that reason, I'm putting this proposal on the backburner when it comes to the Rules Committee's queue - I think it could take a while to write the appropriate language and get it right.

A while back I made a long list of rules for question writing SITs. Some of the points I made are for stylistic consistency within a question set and not appropriate for a best practices document, but I've copy+pasted some relevant details that could make it into a SIT best practices section. Below, for your reading pleasure --

--

3. The question prompts should always begin with "who said it" whenever it's being asked for. If there are three question prompts and "to whom was it said" is one of them, it should always be second. The order can be whatever you like outside of these two rules.

---

5. Do not include SIT questions on quotations introduced with "it is written." These are better written as reference questions. If the text before the quotation is more informative, it can merit inclusion. Example:

Matthew 1:22-23
All this took place to fulfill what the Lord had said through the prophet: "The virgin will conceive and give birth to a son, and they will call him Immanuel" (which means "God with us").

Q: Who said it and how?
A: The Lord had said through the prophet.

--

7. Exact phrases such as "said to him" or "said about him" or "said in Jerusalem" or "said on the fourth day" or "said because" need not be present in the material to ask, respectively, to/about/where/when/why. As long as the text is entirely unambiguous and has only one possible answer, you may ask these questions. Do not use this as an excuse to cram as much information into a question as possible; only add information that is contextually meaningful to the quotation. Common examples for "to whom" include verbs such as replied/asked/answered/retorted, and constructions such as "came to him and said." "About whom" is much rarer, but should be included whenever there are good chances to do so. Lots of examples:

Matthew 8:7-8
Jesus said to him, "Shall I come and heal him?" The centurion replied, "Lord, I do not deserve to have you come under my roof. But just say the word, and my servant will be healed.

Q: Who said it and to whom? "Lord, I do not deserve..."
A: The centurion; to Jesus.

Matthew 8:18-19
When Jesus saw the crowd around him, he gave orders to cross to the other side of the lake. Then a teacher of the law came to him and said, "Teacher, I will follow you wherever you go."

Q: Who said it and to whom?
A: A teacher of the law; to Jesus.

Matthew 11:19
The Son of Man came eating and drinking, and they say, 'Here is a glutton and a drunkard, a friend of tax collectors and sinners.' But wisdom is proved right by her deeds."

Q: About whom was it said? "Here is a glutton and a drunkard, a friend of tax collectors and sinners."
A: The Son of Man.
Explanation: Note that this is a rhetorical statement, so "they" has no direct referent, therefore "who said it" is skipped. It's also a quotation within a quotation, which is acceptable if the nested quotation makes sense on its own as a SIT question.

Matthew 21:12-13
Jesus entered the temple courts and drove out all who were buying and selling there. He overturned the tables of the money changers and the benches of those selling doves. "It is written," he said to them, "'My house will be called a house of prayer,' but you are making it 'a den of robbers.'"

Q: Who said it, to whom was it said, and where was it said? "It is written, 'My house...'"
A: Jesus; to the money changers and those selling doves; the temple courts.
Explanation: The text does not say "Jesus said to them in the temple courts," but it is abundantly obvious from the context and adds meaning to the question.

--

8. Do not overuse the "how was it said" prompt. Common verbs such as replied/answered/asked should never be a "how." Also, "how" should focus on the manner in which the quotation is presented, not necessarily the attending circumstances of the situation.

This is difficult to put into words and very much gray-area/up to interpretation. In my judgment, phrases such as "came to him and said" are not great for "how" because they only describe what the person did before talking, not the way they actually said the quotation.

"Appeared to him in a dream and said" makes more sense for "how" because it tells you the manner of presentation: via dream. "In a dream" could also possibly work as "where was it said" but I think it is much better as "how."

Some things that are commonly written as "how", are better written as "why". Examples from other narrative years:
    "Aware that his disciples were grumbling about this, Jesus said..."
    "The Pharisees, indignant that Jesus was healing was on the Sabbath, said..."
These don't describe the manner of speaking, but rather the circumstance that prompted the speaking. Thus, they are "why was it said."
